src/connect-zbt-1/getting-started/migrate-existing-zha-network.md

```diff
@@ -17,7 +17,57 @@ Follow this guide if you already have a running Zigbee Home Assistant (ZHA) netw
 
 ## To migrate an existing ZHA network to Connect&nbsp;ZBT-1
 
-{% include 'connect-zbt-1/migrate-zha-only.md' %}
+{% steps %}
+
+{% include "connect-zbt-1/plug-in-zbt-1-and-extension-cable.md" %}
+
+{% step "Locating the Home Assistant Connect&nbsp;ZBT-1 integration" %}
+{% image "/static/img/connect-zbt-1/connect-zbt-1-add.png" "Locate the Connect ZBT-1 integration" %}
+{% stepContent %}
+
+- Go to [**Settings** > **Devices & services**](https://my.home-assistant.io/redirect/integrations/).
+- The Connect ZBT-1 should now be discovered.
+- Select **Add**.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Picking your protocol" %}
+{% image "/static/img/connect-zbt-1/zbt-migrate-zigbee-to-new-adapter.png" "Pick your firmware" %}
+{% stepContent %}
+
+- In the dialog, select **Migrate Zigbee to a new adapter**.
+- This will move the Zigbee network to your new adapter.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Select installation type" %}
+{% image "/static/img/connect-zbt-1/zbt-select-installation-method.png" "Pick your firmware" %}
+{% stepContent %}
+
+- **Option 1**: If you just want to migrate, select **Recommended installation**.
+  - This adds the device to Home Assistant and configures Zigbee with the **Zigbee Home Automation** integration that was developed specifically for Home Assistant.
+- **Option 2**: If you have your own controller, for example Zigbee2MQTT, select **Custom**.
+  - This option adds the device to Home Assistant, but you will need to manually install the controller of your choice on your own.
```

Comment:
optional: Do we want to explain what "adds the device to HA" means? (-> we provide firmware updates)
Something like: "This option adds the device to Home Assistant for installing firmware updates, but [...]"?

Reply:
I think it's a bit too much here to explain that. Adding the device to HA is the goal here, to then use whatever functionality comes with it.